packages/lesswrong/server/repos/CommentsRepo.ts

```diff
@@ -609,7 +630,9 @@ class CommentsRepo extends AbstractRepo<"Comments"> {
       shortform: comment.shortform ?? null,
       postedAt: comment.postedAt,
       descendentCount: comment.descendentCount,
-      sources: ['recentComments'],
+      sources: uniq(['recentComments', comment.primarySource ?? 'recentComments']), // temporarily to avoid breaking change, we assign recentComments to all comments
```

Comment:
if the reason for this construction is to avoid pushing `undefined` into the array... I think you should do that with an if statement